pom.xml

```diff
@@ -11,6 +11,7 @@
 
   <properties>
     <keycloak.version>999.0.0-SNAPSHOT</keycloak.version>
+    <infinispan.version>15.0.4.Final</infinispan.version>
```

Comment:
The deployment is currently using 15.0.x Infinispan image, I believe this can break some API compatibility. Is there some nightly maven release?

Reply:
That version is only used by the REST client since the Hot Rod client will use the version shipped with Keycloak. It is fine and only the major version matters.